guides/v2.1/ui_comp_guide/troubleshoot/ui_comp_troubleshoot_js.md

```diff
@@ -64,6 +64,47 @@ var fieldName = ko.contextFor($0).$data;
 console.log(fieldName.name); // product_form.product_form.product-details.container_name.name
 ```
 
+## Debugging using Chrome DevTools
+
+All modern browsers support “debugging” – a special UI in developer tools that makes finding and fixing errors much easier. 
+
+### Sources panel UI
+
+[DevTools] provides a lot of different tools for different tasks, and the **Sources** panel is where we debug JavaScript.
+
+1. Open the required page in Chrome.
+2. Turn on developer tools with F12 (Windows, Linux) or Cmd+Opt+I (Mac).
+3. Select the `Sources` pane.
+
+![Sources Pane]({{ site.baseurl }}/common/images/debugging-sources-pane.png)
+
+In the previous image, we can see three zones:
+
+1. The `Resources` zone lists all the files as HTML, JavaScript, CSS.
+2. The `Source` zone shows the source code of any selected file.
+3. The `Information and control` zone is for debugging.
+
+### Breakpoints
+
+A **breakpoint** is a point of code where the debugger will automatically pause the JavaScript execution process.
```

Comment:
I would say it's a line of code instead of point of code

Reply:
@rogyar the suggestion has been applied.
Thank you.